suite-native/module-home/src/components/PortfolioGraph.tsx

```diff
@@ -1,30 +1,45 @@
-import React, { useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 
-import { Graph, TimeSwitch } from '@suite-native/graph';
-import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { Box, Text } from '@suite-native/atoms';
-import { enabledNetworks } from '@suite-native/config';
-import { Icon } from '@trezor/icons';
-import { selectFiatCurrency } from '@suite-native/module-settings';
-import { useFormatters } from '@suite-common/formatters';
+import { useAtom } from 'jotai';
```

Comment:
why are we using this library?

Reply:
Honestly I always want to try it and it's amazing :D 

But for real now. It was easier way how manage state for these atomic components and squeeze maximum performance from it. Other options could be:

1. React context - will work somehow but it will require much more boilerplate and extra work to make sure to only rerender things we want, because context lot of times triggers unintentional rerenders when things are not memoized perfectly.
2. Redux - It will require whole new reducer just for this, then every change of point will "change" whole State, so it will trigger shallow eq in every useSelector hook in whole app, which will won't cause rerenders in the end but it will require some extra computation power for these shallow eq. Probably other tweaks will be needed to make it really fast and will be very easy to break this optimization with some unintentional changes. I am not talking about how much boilerplate required for such a simple thing...

Jotai in the other hand is small zero-dependency library, which makes things super fast out-of-the-box and it's very hard to make it slow (opposite to redux). It's also widely used these times and it's quite popular so there is minimal chance of supply chain attack.